### ISSUE TITLE
404 on running npx sass-module-types

### ISSUE THREAD

**Issue by pikilon** [babakness/sass-module-types#2]
I am on windows 10 running git bash and I have this result on running the instructions in your package

```
npm ERR! code E404
npm ERR! 404 Not Found: sass-module-types@latest
```

Thank you for your effort, this is a must to feature

**Comment by babakness** [comment]
It may have to do with the previous issue (symlinks). Are you in the "Windows Subsystem for Linux" on windows? 

https://docs.microsoft.com/en-us/windows/wsl/install-win10

I'm not directly familiar with this, but it seems like you can pretty much run Ubuntu on Windows as a sub-system.

**Comment by pikilon** [comment]
No I am running `git bash` which gives me what I need.
I will try with Linux subsystem.

Anyway is good that the package runs in every system.
Let me know if I can help

**Comment by RoystonS** [comment]
I've just got the same issue with a clean install on Windows, v0.0.27.
The problem is that the bin\ directory in the installed package only contains a 0-byte `sass-module-types.d.ts` file and no other files.

I've not got time to do a full test right now, but would modifying the `bin` entry in `package.json` as follows fix the problem by telling npm where the actual file is, avoiding the symlink entirely?

```
  "bin": {
    "sass-module-types": "./lib/sass-module-types.js"
  },
```

**Comment by babakness** [comment]
@RoystonS, thanks for the tip, try `0.0.28` and see if that does it!

**Comment by RoystonS** [comment]
0.0.28 no better for me. I _think_ it needs to be pointing at `./lib/sass-module-types.js`, not just `./lib/sass-module-types`

**Comment by RoystonS** [comment]
Have added a PR for the fixup, after testing it.

**Comment by babakness** [comment]
Thanks for the PR! Published, `0.0.29`
